### PR TITLE
feat(route): 增加Issue Hunt订阅

### PR DESCRIPTION
**Pull request by running-grass**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/issuehunt/funded/DIYgod/RSSHub
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/programming.md [hunk 1]
```diff
@@ -461,6 +461,12 @@ GitHub 官方也提供了一些 RSS:
 
 <Route author="hellodword" example="/hex-rays/news" path="/hex-rays/news"/>
 
+## Issue Hunt
+
+### 项目悬赏
+
+<Route author="running-grass" radar="1" example="/issuehunt/funded/DIYgod/RSSHub" path="/funded/:username/:repo" :paramsDesc="['Github用户/组织','仓库名称']" />
```
[the remark this answers is not in the document]
Comment (running-grass) [reply docs/programming.md:468]: already

File: lib/v2/issuehunt/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/funded/:repo': ['running-grass'],
```
Comment (TonyRL) [inline lib/v2/issuehunt/maintainer.js:2]:
```suggestion
    '/funded/:username/:repo': ['running-grass'],
```
The path here should follow router.js and `path` in docs

File: lib/v2/issuehunt/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'issuehunt.io': {
+        _name: 'Issue Hunt',
+        '.': [
+            {
+                title: '悬赏',
+                docs: 'https://docs.rsshub.app/programming.html#issue-hunt-xiang-mu-xuan-shang',
+                source: '/r/:username/:repo',
+                target: ({ username, repo }) => `/issuehunt/${username}/${repo}`,
```
Comment (TonyRL) [inline lib/v2/issuehunt/radar.js:9]:
```suggestion
                target: ({ username, repo }) => `/issuehunt/funded/${username}/${repo}`,
```
The path should follow router.js

File: docs/programming.md (hunk 1 above)
Comment (TonyRL) [inline docs/programming.md:468]:
```suggestion
<Route author="running-grass" radar="1" example="/issuehunt/funded/DIYgod/RSSHub" path="/issuehunt funded/:username/:repo" :paramsDesc="['Github用户/组织','仓库名称']" />
```

File: lib/v2/issuehunt/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'issuehunt.io': {
+        _name: 'Issue Hunt',
+        '.': [
+            {
+                title: '悬赏',
```
Comment (TonyRL) [inline lib/v2/issuehunt/radar.js:6]:
```suggestion
                title: '项目悬赏',
```
The title here should be the same as H3 header of the docs

File: docs/programming.md [hunk 5]
```diff
@@ -461,6 +461,12 @@ GitHub 官方也提供了一些 RSS:
 
 <Route author="hellodword" example="/hex-rays/news" path="/hex-rays/news"/>
 
+## Issue Hunt
+
+### 项目悬赏
+
+<Route author="running-grass" radar="1" example="/issuehunt/funded/DIYgod/RSSHub" path="/issuehunt funded/:username/:repo" :paramsDesc="['Github用户/组织','仓库名称']" />
```
Comment (TonyRL) [inline docs/programming.md:468]:
```suggestion
<Route author="running-grass" radar="1" example="/issuehunt/funded/DIYgod/RSSHub" path="/issuehunt/funded/:username/:repo" :paramsDesc="['Github用户/组织','仓库名称']" />
```

**Pull request merged**